lookup_tables/ipinfo/ipinfo_location.yml

```diff
@@ -132,6 +132,9 @@ LogTypeMap:
     - LogType: Crowdstrike.UserLogonLogoff
       Selectors:
         - 'aip'
+    - LogType: Crowdstrike.FDREvent
+      Selectors:
+        - 'p_any_ip_addresses'
```

Comment:
We just added new ipinfo lookup tables for data (`ipinfo_asn_datalake.yml` and `ipinfo_location_datalake` would you please update those as well?

Reply:
oh, nice catch. Done. Thank you. https://github.com/panther-labs/panther-analysis/pull/648/commits/2fde01fcf2b41491eafe5a816650949d161814ff